client/coral-plugin-commentbox/CommentBox.js

```diff
@@ -1,6 +1,7 @@
 import React, {Component, PropTypes} from 'react';
 import {I18n} from '../coral-framework';
 import translations from './translations.json';
+import {Button} from 'coral-ui';
```

Comment:
No, it's not published. It's in the 'coral-ui' root folder. These are the basic needed components that go with the Coral Styleguide, so in the future we can add something like, customStyles and modify them all by just importing that. (like we did with Cay)